packages/core/content-manager/admin/src/history/services/historyVersion.ts

```diff
@@ -1,9 +1,17 @@
+import { Data } from '@strapi/types';
+
 import {
   GetHistoryVersions,
   RestoreHistoryVersion,
 } from '../../../../shared/contracts/history-versions';
+import { COLLECTION_TYPES } from '../../constants/collections';
 import { contentManagerApi } from '../../services/api';
 
+interface RestoreVersion extends RestoreHistoryVersion.Request {
+  documentId: Data.ID;
+  collectionType?: string;
+}
```

Comment:
What's the reason to put these here and not in `RestoreHistoryVersion.Request`?

Reply:
These are only used for invalidating the cache and are never sent to the server so it made sense to me to keep them out of the contract and only in the FE scope. What do you think, let me know if you see it differently